FfiSpike/FfiSpike.cpp

```diff
@@ -581,9 +596,9 @@ bool start_afc_client(std::string& device_identifier, std::string& destination,
 
 afc_connection *get_afc_connection(std::string& device_identifier, const char* application_identifier, std::string& root_path, std::string& method_id)
 {
-	if (devices.count(device_identifier) && devices[device_identifier].afc_conn_p)
+	if (devices.count(device_identifier) && devices[device_identifier].afc_connections[application_identifier])
```

Comment:
We do not need this check `devices.[device_identifier].afc_connections.count(application_identifier)` because this `devices[device_identifier].afc_connections[application_identifier]` will not throw an exception if the key does not exist. It will return nullptr and the if will work as expected with the pointer.